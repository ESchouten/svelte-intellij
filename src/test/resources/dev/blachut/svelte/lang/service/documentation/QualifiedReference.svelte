<script lang="ts">
  let local = 323;
</script>

{local.<caret>toFixed()}