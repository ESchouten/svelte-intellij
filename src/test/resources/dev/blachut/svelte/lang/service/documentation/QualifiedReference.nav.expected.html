(method) <a href="https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Number">Number</a>.toFixed(fractionDigits?: number | undefined): string<br/>[ide]<br/>(function) <span style="color:#7a7a43;"><a href="https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Number">Number</a>.toFixed</span><span style="">(fractionDigits?:&#32;</span><span style="color:#000080;">number</span><span style="">):&#32;</span><span style="color:#000080;">string</span> <span style="">(lib.es5.d.ts)</span>